builder/builder-next/exporter/export.go

```diff
@@ -29,11 +29,15 @@ type Differ interface {
 	EnsureLayer(ctx context.Context, key string) ([]layer.DiffID, error)
 }
 
+type ImageTagger interface {
+	TagImageWithReference(imageID image.ID, newTag reference.Named) error
```

Comment:
For others; master diverged; on master, this is;

```suggestion
	TagImage(ctx context.Context, imageID image.ID, newTag distref.Named) error
```